mod_reforged/hooks/items/accessory/wardog_item.nut

```diff
@@ -0,0 +1,20 @@
+::mods_hookExactClass("items/accessory/wardog_item", function(o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+		this.m.StaminaModifier = -3;
+	}
+
+	o.getStaminaModifier <- function()
+	{
+		if (this.isUnleashed())
```

Comment:
This is a perfect use case for the ternary operator:
```squirrel
return this.isUnleashed() ? 0 : this.accessory.getStaminaModifier();
```
Similarly in the other two files.